packages/frontend/src/hooks/useEcharts.ts

```diff
@@ -253,6 +260,10 @@ export const getEchartsSeries = (
                     tooltip: {
                         valueFormatter: valueFormatter(xField, yField, explore),
                     },
+                    color: series.color
```

Comment:
@rephus I made a mistake before!

I meant to say we can set the colours at the very top level: https://github.com/lightdash/lightdash/blob/e8abb8a1cf163d07b0ae16ead2cc047a0c10b422/packages/frontend/src/hooks/useEcharts.ts#L486-L486

Then here we don't need any reference to the organization colours on the individual series.

Reply:
See here: https://echarts.apache.org/en/option.html#color

Reply:
Right, sorry, I thought that was the right configuration. Is there a place were we set the global echart `setOption` config ?

Reply:
Yes! @rephus you can find the actual Echarts call here and the options are passed directly as the output of the `useEcharts` hook: https://github.com/lightdash/lightdash/blob/a7f59f5dc2a3ac12a195fb5fbc8bc140741d77d2/packages/frontend/src/components/SimpleChart/index.tsx#L98-L111